### ISSUE TITLE
The empty inbox background image has a thin colored line at the bottom

### ISSUE THREAD

**Issue by russelldc** [boukestam/inbox-in-gmail#17]
<img width="1636" alt="Screen Shot 2019-04-08 at 2 48 21 PM" src="https://user-images.githubusercontent.com/5100126/55729306-cfddb900-5a0d-11e9-8f05-43abfe15fb27.png">

There's a thin gray(?) line at the very bottom of the sun image. It might be a shadow or a border, not quite sure.

**Comment by russelldc** [comment]
Found the source of the issue, it's a border:

```css
.UI .TB {
    border-bottom: 1px solid rgba(100,121,143,0.122);
}
```

This class only seems to be present when there's no emails on the page, so it may be good enough to just add a border-bottom: none on this class via the extension.

**Comment by russelldc** [comment]
Solved in PR #18
